packages/api/src/aggregate/Just.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type Just<Z extends keyof V, V> =
```

Comment:
Will want to use this in other places as well(like rdp's), so maybe a new folder for this? Also not sure if the rest of these belong in "aggregate" semantically, we could flup this to move to a new folder but also not a big deal